docs/manual/java/guide/build/LagomBuild.md

```diff
@@ -83,7 +83,7 @@ We also recommend using Maven dependency management in your root project pom to
 
 #### A note on Scala versions
 
-When adding dependencies to Lagom libraries, you need to ensure that you include the Scala major version in the artifact ID, for example, `lagom-javadsl-api_2.11`.
+When adding dependencies to Lagom libraries, you need to ensure that you include the Scala major version in the artifact ID, for example, `lagom-javadsl-api_2.12`.
```

Comment:
Maybe we should encourage use of the `scala.binary.version` property in the main doc as well as the migration guide (especially since the artifact uses it).

Reply:
I considered that option for the docs, but kind of give up because many fragments don't have the full `pom.xml` and a reference to a variable without context could be confusing. 

Do you think it will be ok if we add a note recommending the usage of `scala.binary.verison` variable and use it everywhere in the docs? 

I didn't have that issue before because we were not cross-compiling, but now we do.